pkg/controller/node/nodecontroller.go

```diff
@@ -193,12 +202,32 @@ func NewNodeController(
 	return nc
 }
 
+// generateNewAvailabeCIDRs does generating new Available CIDRs index and insert
+// them into availableCIDRs set. Everytime it will generate 256 new CIDRs,
+// once there are no more CIDRs in the network, return false
+func (nc *NodeController) generateNewAvailabeCIDRs() bool {
+	if nc.firstAvailable+256 > nc.maxCIDRs {
```

Comment:
I mean you are right in this case. Since we do not have much chance to reach using that many CIDRs, I just let some CIDRs being wasted